spec/features/publish/courses/new_visa_sponsorship_spec.rb

```diff
@@ -107,4 +109,12 @@ def when_i_choose_yes_for_skilled_worker_visa
   def then_i_should_be_back_on_the_publish_course_confirmation_page
     expect(page).to have_text("Check your answers")
   end
+
+  def and_i_select_no_student_visa
+    publish_courses_new_student_visa_sponsorship_page.no.click
+  end
+
+  def and_i_select_no_skillled_worker_visa
+    publish_courses_new_skilled_worker_visa_sponsorship_page.no.click
+  end
```

Comment:
The publish team is moving away from using these page objects, so I think these should be more like:
```suggestion
  def and_i_select_no_student_visa
     choose 'No'
  end

  def and_i_select_no_skillled_worker_visa
      choose 'No'
  end
```